pkg/machines/app.jsx

```diff
@@ -59,13 +107,32 @@ class App extends React.Component {
                 { config.provider.name === 'LibvirtDBus' && this.state.activeTab == 1 &&
                 <AggregateStatusCards networks={networks} storagePools={storagePools} changeActiveList={this.changeActiveList} />
                 }
+                {Object.keys(this.state.notifications).length > 0 &&
+                <section className="toast-notification-wrapper">
+                    <ToastNotificationList>
+                        {Object.keys(this.state.notifications).map(notificationId => {
```

Comment:
This isn't necessarily a stable order, so it could happen that deleting a notification causes the other notifications to rearrange. Thus some sorting of the property list couldn't hurt. However, this is just a defensive measure and the tests seem to work everywhere.